<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransparencyWidget</class>
 <widget class="QWidget" name="TransparencyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>143</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>403</width>
    <height>143</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="transparencyBox">
     <property name="maximumSize">
      <size>
       <width>391</width>
       <height>85</height>
      </size>
     </property>
     <property name="title">
      <string>Прозрачность</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2"/>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="applianceBox">
       <property name="toolTip">
        <string>Список слоев</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="lockButton">
       <property name="maximumSize">
        <size>
         <width>41</width>
         <height>34</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Заблокировать изменения</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
